tests/test_tokenizer_indexer.py

```diff
@@ -0,0 +1,84 @@
+import sys
+import os
+import torch
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.dirname(SCRIPT_DIR))
+
+# import pytest
+import unittest
+from allennlp.common.testing import AllenNlpTestCase
+from allennlp.common import cached_transformers
+from gector.tokenizer_indexer import PretrainedBertIndexer
+from allennlp.data.tokenizers.whitespace_tokenizer import WhitespaceTokenizer
+from allennlp.data import Vocabulary
+
+class TestPretrainedTransformerIndexer(unittest.TestCase):
```

Comment:
Please add comments/docstrings to various tests to explain what they are testing.